explainer.md

```diff
@@ -622,19 +614,24 @@ partial interface Navigator {
   readonly attribute VR vr;
 };
 
-[SecureContext, Exposed=Window] interface VR : EventTarget {
-  attribute EventHandler ondeviceconnect;
-  attribute EventHandler ondevicedisconnect;
+dictionary VRDeviceFilter {
+  boolean exclusive = false;
+};
+
+dictionary VRDeviceRequestOptions {
+  required sequence<VRDeviceFilter> filters;
```

Comment:
they're simple, but can we add a sample code block showing usage of filters?

Reply:
why's this `required` if it's [`optional` as an argument](https://github.com/w3c/webvr/pull/289/files#diff-817a23c792520ce222813d3eff70d031R627)?

Reply:
Agreed that we need example code that uses filters, I'll try to add that. As for the required on an optional, it's a pattern I've seen elsewhere before. Simply saying that the dictionary itself is optional, but if you include one then you must specify filters. (Otherwise why are you passing the dictionary at all?)

Reply:
After discussion with the rest of the implementers we've decided to remove the filter criteria in the initial version so that we can have more detailed discussions about what's appropriate to filter on before we add it back in down the road. As such the need for example code is now moot. 😉